### PR TITLE
Identity Matrix Checker

### PR DESCRIPTION
**Pull request by HiraIjaz**

### CODE REVIEW

File: 5_IDENTITY matrix Check.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+# (A square matrix is a matrix in which the number of rows 
+# is equal to the number of columns)
+
+import numpy as np
+
+
+def is_identity_matrix(matrix):
+
+    size = len(matrix)
+    identityMat = np.eye(size)
```
Comment (tasawernawaz) [inline 5_IDENTITY matrix Check.py]: rename this to `lower_case`

File: 5_IDENTITY matrix Check.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+
+def is_identity_matrix(matrix):
+
+    size = len(matrix)
+    identity_mat = np.eye(size)
+    if np.array_equal(np.array(matrix), identity_mat):
+        return True
+    else:
+        return False
```
Comment (tasawernawaz) [inline 5_IDENTITY matrix Check.py]: can we do this `Return np.array_equal(np.array(matrix), identity_mat)`

**Review by tasawernawaz — CHANGES_REQUESTED** [review]

File: 5_IDENTITY matrix Check.py (hunk 2 above)
Comment (HiraIjaz) [reply 5_IDENTITY matrix Check.py]: sure

File: 5_IDENTITY matrix Check.py (hunk 1 above)
Comment (HiraIjaz) [reply 5_IDENTITY matrix Check.py]: done

**Review by tasawernawaz — APPROVED** [review]

**Pull request merged**